cmd/pipecd/ops.go

```diff
@@ -82,6 +85,34 @@ func (s *ops) run(ctx context.Context, t cli.Telemetry) error {
 		}
 	}()
 
+	fs, err := createFilestore(ctx, cfg, t.Logger)
+	if err != nil {
+		t.Logger.Error("failed to create filestore", zap.Error(err))
+		return err
+	}
+	defer func() {
+		if err := fs.Close(); err != nil {
+			t.Logger.Error("failed to close filestore client", zap.Error(err))
+		}
+	}()
+
+	// Starting a cron job for insight collector.
+	if s.enable {
+		collector := insightcollector.NewInsightCollector(ds, fs, t.Logger)
+		c := cron.New(cron.WithLocation(time.UTC))
+		_, err := c.AddFunc(cfg.InsightCollector.Schedule, func() {
+			if err := collector.CollectProjectsInsight(ctx); err != nil {
+				t.Logger.Error("failed to run the project insight collector", zap.Error(err))
+			}
+			if err := collector.CollectApplicationInsight(ctx); err != nil {
+				t.Logger.Error("failed to run the application insight collector", zap.Error(err))
+			}
```

Comment:
Btw, Could you add a log to show how long does each function take to run?

Reply:
OK, added:+1: